src/state/atoms/releaseAtom.ts

```diff
@@ -1,5 +1,18 @@
+import axios from 'axios';
+import { updateVisibilityFunctionsBeta, visibilityFunctionsExist } from '../../utils/VisibilitySingleton';
 import { atomWithToggle } from './utils';
+import { getUnleashClient, unleashClientExists } from '../../components/FeatureFlags/unleashClient';
 
-import { isBeta } from '../../utils/common';
-
-export const isPreviewAtom = atomWithToggle(isBeta());
+export const isPreviewAtom = atomWithToggle(undefined, (isPreview) => {
+  // Required to change the `isBeta` function return value in the visibility functions
+  if (visibilityFunctionsExist()) {
+    updateVisibilityFunctionsBeta(isPreview);
+    axios.post('/api/chrome-service/v1/user/update-ui-preview', { uiPreview: isPreview });
```

Comment:
@Hyperkid123 should we have some sort of error handling here?

Reply:
We should. I'll update the PR